erpnext/accounts/doctype/loyalty_program/loyalty_program.py

```
@@ -119,12 +119,12 @@ def get_loyalty_program_details_with_points(
 
 @frappe.whitelist()
 def get_loyalty_program_details(
-	customer,
-	loyalty_program=None,
-	expiry_date=None,
-	company=None,
-	silent=False,
-	include_expired_entry=False,
+	customer: str,
+	loyalty_program: str | None = None,
+	expiry_date: str | None = None,
```

Ruff (0.15.2)

[warning] Unused function argument: `expiry_date`

(ARG001)